### PR TITLE
[2.5] Cherry pick Update documentation for Dockerfile, add location of tbevents, fix link

### PR DESCRIPTION
**Pull request by nvkevlu**
Update documentation for Dockerfile, add location of tbevents, fix link.

### Description

Cherry pick the updates to documentation for Dockerfile, add location of tbevents, fix link.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: docs/resources/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:24.03-py3
+ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:24.07-py3
 FROM ${PYTORCH_IMAGE}
 
-ARG NVF_VERSION=2.4
+ARG NVF_VERSION=main
```
Comment (YuanTingHsieh) [inline docs/resources/Dockerfile:4]: does this need to be 2.5 for 2.5 branch?

Comment (nvkevlu) [reply docs/resources/Dockerfile]: good point, I have updated it

**Review by YuanTingHsieh — APPROVED** [review]
LGTM, we need follow up PRs to upgrade the dockerfile to python3.12

**Pull request merged**